content/docs/manage/demographics.md

```diff
@@ -0,0 +1,67 @@
++++
+title = "Demographics"
+description = "Demographic indicators"
+date = 2022-05-17T16:20:00+00:00
+updated = 2022-05-17T16:20:00+00:00
+draft = false
+weight = 3
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+Demographics is available only on the [Open mSupply Central Server](/docs/getting_started/central). This is where you can view and manage future projections for different demographic indicators.
```

Comment:
is? are? darn it - is this a plural noun or a menu section which is singular??
Perhaps `The demographics section is only...` 🤷